src/validationinterface.h

```diff
@@ -33,7 +34,9 @@ void UnregisterAllValidationInterfaces();
 class CValidationInterface {
```

Comment:
Ahh, I see the issue. Yes, this is actually referring to something further in the change-set, but not in this PR (the full changeset is at https://github.com/TheBlueMatt/bitcoin/commits/2017-01-wallet-cache-inmempool, this is referring to https://github.com/TheBlueMatt/bitcoin/commit/71683dc0d8a00e16211194aaa66bfea1ae265f45).